addons/web/static/tests/helpers/test_utils.js

```diff
@@ -23,6 +23,21 @@ var testUtilsMock = require('web.test_utils_mock');
 var testUtilsModal = require('web.test_utils_modal');
 var testUtilsPivot = require('web.test_utils_pivot');
 
+function deprecatedDomUtils(fn) {
```

Comment:
in web.utils:

function deprecated(fn, message) {
    return function () {
        console.warn(fn.name} + ' is deprecated. ' + (message || '');
        fn.apply(this, arguments);
    };
}